gitlab/v4/objects/namespaces.py

```diff
@@ -20,3 +23,25 @@ class NamespaceManager(RetrieveMixin, RESTManager):
 
     def get(self, id: Union[str, int], lazy: bool = False, **kwargs: Any) -> Namespace:
         return cast(Namespace, super().get(id=id, lazy=lazy, **kwargs))
+
+    @cli.register_custom_action("NamespaceManager", ("namespace", "parent_id"))
```

Comment:
I only defined the optionals for CLI help, not in explicit args, so we don't have to manually maintain them. Might be a little different from older custom methods but I think this may be easier long-term.